### ISSUE TITLE
Using the extension only to expose metadata in CKAN 2.8.3

### ISSUE THREAD

**Issue by mdsnor** [kata-csc/ckanext-oaipmh#8]
Dear developers, 
first of all, thank you for your work.
I was looking for an extension suitable for exposing metadata through the OAI-PMH protocol, and yours was the best option I could find. In our case the harvester component is not required for the moment. However, I wish to describe some challenges I had while trying to use it with CKAN 2.8.3 (however, I didn't perform a detailed analysis):

 -  It is not possible to use this extension alone, without other required extensions. for example it requires the ckanext-kata to be installed too, requiring its helpers and even some library dependencies in requirements.txt. And it's a chain reaction, since also the ckanext-kata comes with other dependencies...
- Some of the code is outdated, and not ready for CKAN 2.8.3 (especially in ckanext-kata)

As a workaround, after forking, I first tried to make this extension independent from ckanext-kata, by disabling some of the broken imports (we are using only part of the code), and copying and pasting the helpers from ckanext-kata to ckanext-oaipmh. It works, but following a colleague's suggestion, we opted rather to keep the ckanext-kata to be installed as a library (without enabling any plugin).

Probably it could become more useful, in my opinion, specifically in our case (no harvester needed), to:

- make the extension depending only from really necessary extensions
- maybe isolate the harvesting part from the metadata serving one (also in terms of dependencies)
- Update to latest CKAN version

Thanks!

**Comment by xbian** [comment]
Hi @mdlux 

We are trying to install this extension too, but without a success, could you share your fork please? We would like to try it out too.

Thanks
@xbian

**Comment by bhavin2897** [comment]
Hi @mdsnor 

Thanks for the information and suggestions. 
Could you please share the fork or suggest me any other OAI-PMH harvesters that you have been using in CKAN 2.9? 
This extension (ckan-oaipmh) seems outdated as you have mentioned. 

Thanks in advance.

**Comment by frafra** [comment]
Last five commits of https://github.com/NINAnor/ckanext-kata/commits/nina-version-kata have been used for CKAN 2.8.4.

**Comment by frafra** [comment]
@bhavin2897 are you interested in maintaining this extension?

**Comment by bhavin2897** [comment]
@frafra No. The best version for this extension is [OAI-PMH Harvester branch py3](https://github.com/DataShades/ckanext-oaipmh/tree/py3) 

Maybe it would be great, if developers are looking for this harvester, and directs to latest one (above link) 

Thank you for your concern

**Comment by frafra** [comment]
The problem is that there is no OAI PMH server with the alternative extension.

I found this, instead, https://github.com/EUDAT-B2FIND/ckanext-oaipmh-server, but it seems not working: https://github.com/EUDAT-B2FIND/ckanext-oaipmh-server/issues/11. `ckanext-spatial` has some support for pycsw, but it has been broken for quite a while: https://github.com/ckan/ckanext-spatial/issues/298.
